ui/app/js/services/releases_service.js

```diff
@@ -46,6 +46,19 @@ angular.module("app").factory('Releases', function($http, $q, ScheduledChanges,
         }
       });
     },
+    getUpDiff: function(sc_id) {
+      var url = '/api/scheduled_change/diff/release/' + sc_id;
+      return $http({
+        url: url,
+        method: 'GET',
+        transformResponse: function(value) {
```

Comment:
Huh, this seems very strange given that the response's Content-Type is text/plain - good catch though.